### ISSUE TITLE
freeze_support error when creating a strategy (same as Issue #181)

### ISSUE THREAD

**Issue by voxvici** [twopirllc/pandas-ta#282]
worked on fedora 33 a week ago, same setup, this is fresh windows install
besides small hiccups great work on package real quality work keep it up

**version:** pandas-ta 0.2.45b via pip install pandas-ta
**python version:** 3.9.1
**system:** windows 10 x64
**code:**
```python
import pandas_ta as ta
import pandas as pd

from itertools import islice
from collections import Counter

import yfinance as yf

import datetime

data = yf.download('SPY', period="3mo", interval="60m")
df = pd.DataFrame(data)
df['Date'] = pd.to_datetime(df.index)

MACDStrategy = ta.Strategy(
    name="MACD",
    description="MACD",
    ta=[
        {"kind": "macd", "fast": 12, "slow": 27},
    ]
)

df.ta.strategy(MACDStrategy)

```
**log:**
```sh
    prep_data = spawn.get_preparation_data(process_obj._name)

  File "C:\Python39\lib\multiprocessing\spawn.py", line 154, in get_preparation_data
    _check_not_importing_main()
  File "C:\Python39\lib\multiprocessing\spawn.py", line 134, in _check_not_importing_main
    raise RuntimeError('''
RuntimeError:
        An attempt has been made to start a new process before the
        current process has finished its bootstrapping phase.

        This probably means that you are not using fork to start your
        child processes and you have forgotten to use the proper idiom
        in the main module:

            if __name__ == '__main__':
                freeze_support()
                ...

        The "freeze_support()" line can be omitted if the program
        is not going to be frozen to produce an executable.
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "C:\Python39\lib\multiprocessing\spawn.py", line 116, in spawn_main
    exitcode = _main(fd, parent_sentinel)
  File "C:\Python39\lib\multiprocessing\spawn.py", line 125, in _main
    prepare(preparation_data)
  File "C:\Python39\lib\multiprocessing\spawn.py", line 236, in prepare
    _fixup_main_from_path(data['init_main_from_path'])
  File "C:\Python39\lib\multiprocessing\spawn.py", line 287, in _fixup_main_from_path
    main_content = runpy.run_path(main_path,
  File "C:\Python39\lib\runpy.py", line 268, in run_path
    return _run_module_code(code, init_globals, run_name,
  File "C:\Python39\lib\runpy.py", line 97, in _run_module_code
    _run_code(code, mod_globals, init_globals,
  File "C:\Python39\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "D:\backup\ad_luna\kraken.py", line 34, in <module>
    df.ta.strategy(MACDStrategy)
  File "C:\Python39\lib\site-packages\pandas_ta\core.py", line 666, in strategy
    pool = Pool(self.cores)
  File "C:\Python39\lib\multiprocessing\context.py", line 119, in Pool
    return Pool(processes, initializer, initargs, maxtasksperchild,
  File "C:\Python39\lib\multiprocessing\pool.py", line 212, in __init__
    self._repopulate_pool()
  File "C:\Python39\lib\multiprocessing\pool.py", line 303, in _repopulate_pool
    return self._repopulate_pool_static(self._ctx, self.Process,
  File "C:\Python39\lib\multiprocessing\pool.py", line 326, in _repopulate_pool_static
    w.start()
  File "C:\Python39\lib\multiprocessing\process.py", line 121, in start
```

**Comment by ketav** [comment]
I am getting the same error on Mac Book Pro Python3.9? Does it mean it is not fixable?

**Comment by twopirllc** [comment]
Hello @voxvici,

> besides small hiccups great work on package real quality work keep it up

Thanks. Please list other issues as separate Issues so I know what to look out for.
Also, Make sure you are always on the Latest Version (_0.2.75b_):
```sh
$ pip install -U git+https://github.com/twopirllc/pandas-ta
```

> worked on fedora 33 a week ago, same setup, this is fresh windows install

Yup. So this is a Python Multiprocessing Issue and **not** a Pandas TA Issue has been resolved in Pinned Issue #181. It's a rather simple fix.

Kind Regards,
KJ
